dev-lisp/roswell/roswell-22.12.14.113.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
Please use pkgdev to commit so it fixes these small things for you automatically :)